1.generate-profiles/1.aggregate.py

```diff
@@ -0,0 +1,92 @@
+import os
+import sys
+import pathlib
+import argparse
+import warnings
+import pandas as pd
+
+from pycytominer import aggregate
+from pycytominer.cyto_utils import output
+
+sys.path.append("../scripts")
+from config_utils import process_config_file
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--config_file",
+    help="configuration yaml file for the profiling pipeline",
+    default="profiling_config.yaml",
+)
+args = parser.parse_args()
+config_file = args.config_file
+
+config = process_config_file(config_file)
+
+# Extract config arguments
+core_args = config["core"]
+batch = core_args["batch"]
+trash_files = core_args["trash_files"]
+float_format = core_args["float_format"]
+compression = core_args["compression"]
+
+single_cell_args = config["single_cell"]
+single_cell_output_dir = single_cell_args["single_cell_output_dir"]
+single_cell_file = single_cell_args["single_file_only_output_file"]
+
+aggregate_args = config["aggregate"]
+aggregate_from_site = aggregate_args["from_site_files"]
+aggregate_from_single_file = aggregate_args["from_single_file"]
+aggregate_output_dir = aggregate_args["aggregate_output_dir"]
+aggregate_operation = aggregate_args["operation"]
+aggregate_features = aggregate_args["features"]
+aggregate_levels = aggregate_args["levels"]
+
+# Input argument flow control
+if aggregate_from_single_file:
+    assert single_cell_file.exists(), "Error! The master single cell file does not exist! Did you mean to set `from_single_file` to False?"
+else:
+    assert aggregate_from_site, "You must set one aggregate source to true in the config (either `from_site_files` or `from_single_file`)"
+
+if aggregate_from_single_file and aggregate_from_site:
+    warnings.warn(
+        "Both sources set to true. Defaulting to aggregate `from_single_file`"
```

Comment:
More clear to say:
"Both `from_site_files` and `from_single_file` are set to true in the config. Defaulting to aggregate `from_single_file`"